### ISSUE TITLE
Fully supervised setting of LBF

### ISSUE THREAD

**Issue by Zhenwen-NLP** [evelinehong/LBF#1]
Could you please provide the training code which initializes the memory buffer with all the ground-truth expressions? I have tried to re-produce, but I cannot reach 74.1 which is reported in your paper.

**Comment by evelinehong** [comment]
Hi,
I'm sorry that I cannot find the original codes. As I remember, I tried several things:
1) Pretrain the model on the annotated expressions
2) Initialize the memory buffer with all the ground-truth expressions, and sample a few expressions (like in MAPO) by LBF
3) Assign different hyperparameters to the loss of fully-supervised part (on ground-truth expressions), as well as the loss of LBF part (on the fixed trees we find)

Best,
Yining
